src/press_quote/quote.macros.html

```diff
@@ -0,0 +1,15 @@
+{#
+  * @overview Macros for press quotes.
+#}
+
+{% from '../icons/icons.macros.html' import svg %}
+
+{#
+  * Renders a single press quote with source logo beneath.
+#}
+{% macro press_quote(content, logo, class=null, id=null) %}
+<div class="press-quote {% if class %}{{ class }}{% endif %}" {% if id %}id="{{ id }}"{% endif %}>
```

Comment:
Since we'll want to add `class` and `id` to all of our macros, I would recommend creating a `utilities.macros.html` file. We can make a macro that adds `{% if class %}{{ class }}{% endif %}" {% if id %}id="{{ id }}"{% endif %}`